frontend/src/components/lib/components/BaseInput/BaseInput.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactNode } from 'react';
```

Comment:
Please put this file on the right dir under components/base (this is why this folder still exist)